src/dos/program_more_output.h

```diff
@@ -39,6 +39,8 @@ class MoreOutputBase {
 	virtual void Display() = 0;
 
 protected:
+	static uint8_t GetCurrentColumn();
+	static uint8_t GetCurrentRow();
```

Comment:
Hmm, so are these zero-indexed? (I assume yes)
Is "current" the position where the last character was written to, or where the next character would land at? If the latter, I'm guessing the current row gets auto-incremented when the end of the line has been reached?
I think some explanation is in order for these for the benefit of future maintainers.

Reply:
Changed names to `GetCursorColumn` and `GetCursorRow`. Hopefully it is more clear now what these functions are doing.

Reply:
> Changed names to `GetCursorColumn` and `GetCursorRow`. Hopefully it is more clear now what these functions are doing.

Yes, definitely it's clear now!